src/generator.csproj

```diff
@@ -145,6 +145,7 @@
     <Compile Include="..\tools\common\NullableAttributes.cs">
       <Link>NullableAttributes.cs</Link>
     </Compile>
+    <Compile Include="bgen\Api.cs" />
```

Comment:
Would it make sense for the Api class to be in the Models folder?

Reply:
yep, since the PR has a merge conflict, I'll move it there.